documentation/man/features/config.rst

```diff
@@ -57,3 +60,7 @@ If you want to display all configurations you could use::
 If you want to display deploy configurations you could use::
 
   kw config -s deploy
+
+If you want to change a configuration with verbose mode you could use::
```

Comment:
How about something like:
In case of any issue, you can try to enable the verbose option::